mobile/src/base/java/be/mygod/vpnhotspot/EBegFragment.kt

```diff
@@ -52,10 +48,16 @@ class EBegFragment : DialogFragment(), PurchasesUpdatedListener, BillingClientSt
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View =
             inflater.inflate(R.layout.fragment_ebeg, container, false)
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
+        dialog.requestWindowFeature(Window.FEATURE_NO_TITLE)
```

Comment:
Why's this needed?

Reply:
I just checked again and this issue only appears to happen on older Android flavours (was running an emulator with API 21) . Dialog appears normal on my phone running Oreo. I guess you might want to keep that flag